librz/analysis/p/analysis_bf.c

```diff
@@ -86,48 +464,56 @@ static int bf_op(RzAnalysis *analysis, RzAnalysisOp *op, ut64 addr, const ut8 *b
 		free((ut8 *)buf);
 		break;
 	case ']':
+		oplist = bf_rlimit(vm, stack_helper, op->id, addr);
 		op->type = RZ_ANALYSIS_OP_TYPE_UJMP;
 		// XXX This is wrong esil
```

Comment:
Remove ESIL formation from this file. We will aim for brainfuck as a clean simplest example of uplifting to RZIL.

Reply:
Ok I will test and debug this in the next push